geniza/corpus/models.py

```diff
@@ -527,6 +527,20 @@ def iiif_urls(self):
             )
         )
 
+    def iiif_images(self):
```

Comment:
this is more display logic than I'd prefer in the models; we should think about adapting this template tag from mep (or maybe it could be added to `djiffy` for re-use):

https://github.com/Princeton-CDH/mep-django/blob/main/mep/common/templatetags/mep_tags.py#L149-L172